mycroft/client/enclosure/mark1/__init__.py

```diff
@@ -220,7 +223,7 @@ class EnclosureWriter(Thread):
     Note: A command has to end with a line break
     """
 
-    def __init__(self, serial, bus, size=16):
+    def __init__(self, serial, bus, size = 16):
```

Comment:
In function-calls there should be no spaces around `=`.

It's weird but that's the pep8 style :/